bindings/pydeck-carto/pydeck_carto/carto_auth/examples/connector_bigquery.py

```diff
@@ -4,19 +4,19 @@
 
 Use CARTO DW client (BigQuery)
 """
-from pydeck_carto import CartoAuth
+from pydeck_carto.carto_auth.auth import CartoAuth
 
 ca = CartoAuth.from_file("./carto_credentials.json")
 
-cdw_client = ca.get_carto_dw_client()
+bq_client = ca.get_bigquery_client()
```

Comment:
This looks like a revert. Any reason for the change?

Reply:
Looks like a wrong merge with master, fixed